From: Sachin Kamat <sachin.kamat@linaro.org>
Date: Tue, 4 Jun 2013 00:31:20 +0000
Subject: [PATCH 10/10] net: sun4i-emac: Staticize local symbols
Origin: https://git.kernel.org/cgit/linux/kernel/git/davem/net-next.git/commit?id=11a164a04382d735230b01f4cc46ad78a7c4abf6
Bug-Debian: http://bugs.debian.org/711998

Some symbols referenced only in this file are made static.

Signed-off-by: Sachin Kamat <sachin.kamat@linaro.org>
Cc: Stefan Roese <sr@denx.de>
Cc: Maxime Ripard <maxime.ripard@free-electrons.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/ethernet/allwinner/sun4i-emac.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/allwinner/sun4i-emac.c b/drivers/net/ethernet/allwinner/sun4i-emac.c
index e9c8dbe..50b853a 100644
--- a/drivers/net/ethernet/allwinner/sun4i-emac.c
+++ b/drivers/net/ethernet/allwinner/sun4i-emac.c
@@ -258,7 +258,7 @@ static const struct ethtool_ops emac_ethtool_ops = {
 	.get_link	= ethtool_op_get_link,
 };
 
-unsigned int emac_setup(struct net_device *ndev)
+static unsigned int emac_setup(struct net_device *ndev)
 {
 	struct emac_board_info *db = netdev_priv(ndev);
 	unsigned int reg_val;
@@ -310,7 +310,7 @@ unsigned int emac_setup(struct net_device *ndev)
 	return 0;
 }
 
-unsigned int emac_powerup(struct net_device *ndev)
+static unsigned int emac_powerup(struct net_device *ndev)
 {
 	struct emac_board_info *db = netdev_priv(ndev);
 	unsigned int reg_val;
-- 
1.7.10.4

